node_modules/oae-core/topnavigation/js/topnavigation.js

```diff
@@ -67,7 +67,8 @@ define(['jquery', 'oae.core', 'activityadapter'], function($, oae, ActivityAdapt
         var initLeftMenu = function() {
             var $template = oae.data.me.anon ? $('#topnavigation-left-anonymous-template') : $('#topnavigation-left-loggedin-template');
             oae.api.util.template().render($template, {
-                'authStrategyInfo': authStrategyInfo
+                'authStrategyInfo': authStrategyInfo,
+                'redirectUrl': signInRedirectUrl || oae.api.util.url().attr('relative')
```

Comment:
Can we

a. Default to `oae.api.util.url().attr('relative')` inside of oae.api.authentication.getLoginRedirectUrl();`

or

b. Default to `oae.api.util.url().attr('relative')` on line 27 (`var signInRedirectUrl = oae.api.authentication.getLoginRedirectUrl();`

// Get the redirect URL
         var signInRedirectUrl =